velox/exec/HashJoinBridge.cpp

```diff
@@ -64,6 +64,24 @@ void HashJoinBridge::addBuilder() {
   ++numBuilders_;
 }
 
+void HashJoinBridge::reclaim() {
+  std::lock_guard<std::mutex> l(mutex_);
+  if (tableSpillFunc_ == nullptr || probeStarted_.load() ||
+      !buildResult_.has_value() || buildResult_->table == nullptr ||
+      buildResult_->table->numDistinct() == 0) {
+    return;
+  }
+  VELOX_CHECK(buildResult_.has_value());
+  VELOX_CHECK_NOT_NULL(buildResult_->table);
+
+  auto spillPartitionSet = tableSpillFunc_(buildResult_->table);
+  buildResult_->table->clear(true);
+  setSpilledHashTableLocked(spillPartitionSet);
```

Comment:
```
s/setSpilledHashTableLocked/setSpilledHashTablePartitionSetLocked/
```

Reply:
I want to make the name consistent with the public one.